lib/gap_intelligence/client/requestable.rb

```diff
@@ -1,3 +1,5 @@
+require 'uri'
```

Comment:
do we need to require uri here? I see we require it in `lib/gap_intelligence.rb'